src/renderer/components/wallet/MnemonicPhrase.tsx

```diff
@@ -13,23 +14,27 @@ type Props = {
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const MnemonicPhrase = React.forwardRef<any, Props>(({ words, onWordClick, readOnly }, ref) => {
   return (
-    <Row ref={ref}>
+    <Styled.Row ref={ref}>
       <Col span={24}>
-        <Card bodyStyle={{ padding: '6px', minHeight: '100px' }}>
-          <div>
+        <Styled.Card bodyStyle={{ padding: '6px', minHeight: '100px' }}>
```

Comment:
[nitpick] Since we have `Styled.Card` you could move its `bodyStyle` declaration to `*.styles.tx`

```
export const Card = styled(A.Card).attrs({ bodyStyle: { padding: '6px', minHeight: '100px' } })`
...`
```